src/main/viz/IdiogramTrack.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * A track which shows an idiogram corresponding to the genome being
+ * shown by the reference track. 
+ * @flow
+ */
+'use strict';
+
+import type {VizProps} from '../VisualizationWrapper';
+import type {State} from '../types';
+import type {DataSource} from '../sources/DataSource';
+import React from 'react';
+import EmptySource from '../sources/EmptySource';
+import d3 from 'd3';
+import idiogrammatik from '../../../node_modules/idiogrammatik.js/idiogrammatik';
```

Comment:
Can you just import this normally? It is in package.json so you should not have to specify the path